titiler/image/main.py

```diff
@@ -66,6 +67,9 @@
 tiles = LocalTilerFactory(router_prefix="/image")
 app.include_router(tiles.router, tags=["Local Tiles"], prefix="/image")
 
+tiles = GCPSTilerFactory(router_prefix="/map")
```

Comment:
let's make this `geo` and then we can keep `/map` endpoint from the tiler factory